Bottom: 1c1ea36851b79c27de1fb9356d6ca5322d2ea6d2
Top:    4a50b784eabc87d349e0a2751db08456eb571f9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 10:34:54 +0000

Refresh of profile-noholes-dicer-and-add-

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 67f5665..abad25a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -913,6 +913,8 @@ intersect_impl (jmp_buf * jb, POLYAREA * b, POLYAREA * a, int add)
   for (pa = a->contours; pa; pa = pa->next)     /* Loop over the contours of POLYAREA "a" */
     {
       BoxType sb;
+      jmp_buf out;
+      int retval;
 
       c_info.getout = NULL;
       c_info.pa = pa;
